code/game/objects/items/fleece.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/fleece
+	name        = "fleece"
+	desc        = "The shorn fleece of some animal, ready for spinning."
+	icon        = 'icons/obj/items/fleece.dmi'
+	icon_state  = ICON_STATE_WORLD
+	item_flags  = ITEM_FLAG_IS_BELT
+	slot_flags  = SLOT_HEAD | SLOT_LOWER_BODY | SLOT_OVER_BODY
```

Comment:
Is it wearable? There doesn't seem to be onmob icons aside from inhands in the dmi

Reply:
still present :(